README.md

```diff
@@ -1,2 +1,82 @@
-# gmdf
-glTF Metadata File for STK
+# The `*.gmdf` file
+
+The glTF Metadata File for STK (Systems Tool Kit)
+
+## Contributors
+
+* Greg Beatty, Analytical Graphics, Inc.
+* Ed Mackey, Analytical Graphics, Inc.
+* Alex Wood, Analytical Graphics, Inc.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec, with the `AGI_articulations` and `AGI_stk_metadata` extensions applied.
+
+## Overview
+
+The `AGI_articulations` and `AGI_stk_metadata` glTF 2.0 vendor extensions supply a number of different kinds of metadata to help with analysis of physical objects represented by glTF models.  As an alternative to supplying this metadata as extensions within a glTF file, the same metadata can also be supplied in an external file with the `*.gmdf` file extension, with the same path and base filename as the associated `*.gltf` or `*.glb` file.  One purpose of this is to allow the model author to iterate on their model, re-exporting it from a DCC tool or content pipeline to produce new versions of the model without overwriting the existing metadata.  Optionally, once the model is finalized, the metadata can then be moved into the glTF's JSON structure as extensions, and the external file deleted.
```

Comment:
DCC tool?  Dsomething content creation?  Please expand on this acronym.  Also, maybe we call out that gmdf is short for "glTF Meta Data File".

Reply:
Digital Content Creation.  I'll change it.